_overviews/scala3-migration/incompat-type-checker.md

```diff
@@ -18,17 +18,19 @@ It is unsound and might cause runtime failures, as demonstrated by this [test](h
 
 The Scala 3 compiler does not permit this anymore.
 
-```scala
+{% tabs scala-3-unsound_vc_1 %}
+{% tab 'Scala 3 Only' %}
+~~~ scala
 class Foo[-A](x: List[A]) {
   def f[B](y: List[B] = x): Unit = ???
 }
 
 class Outer[+A](x: A) {
   class Inner(y: A)
 }
-```
+~~~
 
-{% highlight text %}
+~~~ text
```

Comment:
Did you check that the rendering is correct? IIRC, we had to use the `highlight` Jekyll directive for some reason.